kvm/src/lib.rs

```diff
@@ -26,11 +28,12 @@ use libc::{open, EINVAL, ENOSPC, O_RDWR};
 use kvm_gen::*;
 use memory_model::MemoryMapping;
 use sys_util::{errno_result, Error, EventFd, Result};
-use sys_util::{
-    ioctl, ioctl_with_mut_ptr, ioctl_with_mut_ref, ioctl_with_ptr, ioctl_with_ref, ioctl_with_val,
-};
+use sys_util::{ioctl, ioctl_with_ref, ioctl_with_val};
+#[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
```

Comment:
Because they are used by functions that call ioctl which are x86 specific. Not labeling them would result in a warning when building on aarch64. It is possible that in the future (when I actually add aarch64 functionality) these ioctl constructs will be used on aarch64 too.